ui/lib/utils/i18n.ts

```diff
@@ -37,18 +35,32 @@ export function addTranslationResource(lang, translations) {
 }
 
 export const ALL_LANGUAGES = {
-  zh: '简体中文',
+  'zh-CN': '简体中文',
```

Comment:
this might cause bugs since all locale files are now named as `zh` instead of `zh-CN`.

Reply:
But actually it works fine after trying. Because all languages that start with 'zh-' will fallback to use 'zh' translation first if they can't find its own language translation, and this is i18next's default behavior. https://www.i18next.com/principles/fallback#locals-resolving